src/main/java/com/pneumaticraft/commandhandler/CommandHandler.java

```diff
@@ -175,6 +178,68 @@ public void registerCommand(Command command) {
         }
     }
 
+    /**
+     * Parse vanilla selectors. See https://minecraft.gamepedia.com/Commands#Target_selectors about it.
+     *
+     * @param sender  Object that executed the command
+     * @param args    Command arguments
+     * @return Arguments list with selectors parsed to playerName if any.
+     */
+    public List<String> parseSelectors(CommandSender sender, List<String> args) {
+        if (sender == null || args == null) {
+            throw new IllegalArgumentException("sender and args cannot be null!");
+        }
+
+        for (ListIterator<String> argsIterator = args.listIterator(); argsIterator.hasNext(); ) {
+            String arg = argsIterator.next();
+            String parsedArg = parseSingleSelector(sender, arg);
+            if (parsedArg.equals(arg)) {
+                continue;
+            }
+            argsIterator.set(parsedArg);
+        }
+
+        return args;
+    }
+
+    /**
+     * Parse a vanilla selector. See https://minecraft.gamepedia.com/Commands#Target_selectors about it.
+     *
+     * @param sender  Object that executed the command
+     * @param arg    A single command argument
+     * @return Arguments list with selectors parsed to playerName if any.
+     */
+    public String parseSingleSelector(CommandSender sender, String arg) {
+        if (sender == null || arg == null) {
+            throw new IllegalArgumentException("sender and arg cannot be null!");
+        }
+        if (!arg.startsWith("@")) {
+            return arg;
+        }
+
+        List<Player> matchedPlayers;
+        try {
+            matchedPlayers = this.plugin.getServer().selectEntities(sender, arg).stream()
+                    .filter(e -> e instanceof Player)
+                    .map(e -> ((Player) e))
+                    .collect(Collectors.toList());
```

Comment:
You're only allowing player matches right now. I guess that's fine but it would be cool if commands could match other entities.